packages/deployer/test/compiler_test.ts

```diff
@@ -38,9 +37,9 @@ describe('#Compiler', function() {
         };
         const exchangeArtifactString = await fsWrapper.readFileAsync(exchangeArtifactPath, opts);
         const exchangeArtifact: ContractArtifact = JSON.parse(exchangeArtifactString);
-        const exchangeContractData: ContractNetworkData = exchangeArtifact.networks[constants.networkId];
+        const exchangeContractData = exchangeArtifact.versions[constants.defaultVersionName];
         // The last 43 bytes of the binaries are metadata which may not be equivalent
-        const unlinkedBinaryWithoutMetadata = exchangeContractData.bytecode.slice(0, -86);
+        const unlinkedBinaryWithoutMetadata = exchangeContractData.compilerOutput.evm.bytecode.object.slice(0, -86);
```

Comment:
Ditto above. No guarentees anymore that we have this available to us **unless** we force users to always include the `bytecode`. Thoughts?